src/config/graphical-session/wayland.md

```diff
@@ -80,15 +80,9 @@ compositors, and is installed as a dependency for most of them. Its package is
 
 ## Configuration
 
-The Wayland library uses the `XDG_RUNTIME_DIR` environment variable to determine
-the directory for the Wayland socket.
-
-Install `elogind` as your [session manager](../session-management.md) to
-automatically setup `XDG_RUNTIME_DIR`.
-
-Alternatively, manually set the environment variable through the shell. Make
-sure to create a dedicated user directory and set its permissions to `700`. A
-good default location is `/run/user/$(id -u)`.
+The Wayland library uses the
```

Comment:
I guess we can improve the text to say "requires" instead of "uses", since it's mandatory.